aospy/utils/io.py

```diff
@@ -38,17 +35,6 @@ def get_parent_attr(obj, attr, strict=False):
             return None
 
 
-def dict_name_keys(objs):
-    """Create dict whose keys are the 'name' attr of the objects."""
-    assert isinstance(objs, (tuple, list, dict, set))
-    if isinstance(objs, (tuple, list, set)):
-        try:
-            return {obj.name: obj for obj in objs}
-        except AttributeError as e:
-            raise AttributeError(e)
-    return objs
-
-
 def to_dup_list(x, n, single_to_list=True):
```

Comment:
I just realized we no longer use this (untested) function, so I think we can remove it